src/main/java/lotto/utils/StringUtils.java

```diff
@@ -0,0 +1,20 @@
+package lotto.utils;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class StringUtils {
+    private static final String DELIMITER = ", ";
+
+    public static List<Integer> parseWithDelimiter(String input) {
+        return Arrays.stream(input.split(DELIMITER))
+            .mapToInt(number -> Integer.parseInt(number))
```

Comment:
람다식의 실행코드가 한줄일때 람다식의 파라미터 갯수 와 실행되는 메소드의 파라미터 갯수가 같으면 메소드 참조 형식으로 쓸 수 있어요.
```suggestion
            .mapToInt(Integer::parseInt)
```

Reply:
넵 심지어 인텔리제이가 힌트를 줬는데도 지나쳤더라고요...ㅜㅜ 반성합니다.
알려주셔서 감사합니다!